frontend/src/features/projects/list/ProjectListView.tsx

```diff
@@ -137,33 +167,48 @@ const ProjectListView: React.FC = () => {
     setDeleteId(projectNumber);
   };
 
+  const onRowClick = (row: IProject) => {
+    if (mode === PageMode.APPROVAL) {
+      history.push(`/dispose/projects/assess/properties?projectNumber=${row.projectNumber}`);
+    } else {
+      history.push(
+        `/dispose/projects/${row.status.toLowerCase()}?projectNumber=${row.projectNumber}`,
+      );
+    }
+  };
   return (
     <Container fluid className="ProjectListView">
       <div className="filter-container">
-        <FilterBar<IProjectFilterState>
-          initialValues={filter}
-          onSearch={values => setFilter(values)}
-          onReset={() => setFilter({})}
-        >
-          <Col>
-            <h4>My Agency's Projects</h4>
-          </Col>
-          <Col className="bar-item">
-            <Form.Group>
-              <Form.Label>Active Projects:&nbsp;</Form.Label>
-              <Field name="active" label="Active Projects" type="checkbox" />
-            </Form.Group>
-          </Col>
-          <Col className="bar-item">
-            <Form.Group>
-              <Form.Label>My projects:&nbsp;</Form.Label>
-              <Field name="createdByMe" type="checkbox" />
-            </Form.Group>
-          </Col>
-          <Col className="bar-item">
-            <Input field="name" placeholder="Search by project name" />
-          </Col>
-        </FilterBar>
+        {filterable ? (
+          <FilterBar<IProjectFilterState>
+            initialValues={filter}
+            onSearch={values => setFilter(values)}
+            onReset={() => setFilter({})}
+          >
+            <Col>
+              <h4>{title}</h4>
+            </Col>
+            <Col className="bar-item">
+              <Form.Group>
+                <Form.Label>Active Projects:&nbsp;</Form.Label>
+                <Field name="active" label="Active Projects" type="checkbox" />
+              </Form.Group>
+            </Col>
+            <Col className="bar-item">
+              <Form.Group>
+                <Form.Label>My projects:&nbsp;</Form.Label>
```

Comment:
When I tested this filter for the standard **Project List View** it did not work as expected.  I'll post a bug once I have some formal time to test again.